admin/services/check-form.service.js

```diff
@@ -297,7 +297,7 @@ const checkFormService = {
     result = csvData.toString().split('\n').map(function (line) {
       return line.trim()
     }).filter(Boolean)
-    return result.length === config.LINES_PER_CHECK_FORM
+    return result.length === parseInt(config.LINES_PER_CHECK_FORM)
```

Comment:
i would consider moving the parseInt into the config file instead.  and return default value if parsing fails